### ISSUE TITLE
get_viewport().get_mouse_position() do not update while moving mouse.

### ISSUE THREAD

**Issue by stmSi** [godotengine/godot#55716]
### Godot version

v4.0.dev.custom_build [46d384060]

### System information

5.13.19-2-MANJARO XFCE, Nvidia 2060

### Issue description

get_viewport().get_mouse_position() do not update while moving mouse. 
get_mouse_position() will only update if Left/Middle/Right clicking inside viewport.
```
func _process(delta: float) -> void:
    print(get_viewport().get_mouse_position())
```

### Steps to reproduce

1. Add Any Node
2. Attach Script
3. Test Following Code
```
func _process(delta: float) -> void:
	print('node2d ', get_viewport().get_mouse_position())
```

### Minimal reproduction project

[test.zip](https://github.com/godotengine/godot/files/7674387/test.zip)

**Comment by stmSi** [comment]
my current workaround hack is 
`DisplayServer.mouse_get_position() - DisplayServer.window_get_position()` 
until it fix 😛

**Comment by jmb462** [comment]
Cannot reproduce this issue neither with Windows 10 nor with Ubuntu 21.10
Mouse coordinate seems to update correctly without clicking with the MRP.

**Comment by stmSi** [comment]
I found a weird behaviour. When you Resize/Alt-Tab/Minimize-Maximize the game window, It starts to work.

**Comment by stmSi** [comment]
During the testing in v4.0.alpha.calinou [b480140ce], the issue is fixed. It may be my OS problem. Closing this issue.

**Comment by akien-mga** [comment]
This was a duplicate of #53894, fixed by #56548.
